src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  height: 408px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  padding-top: 32px;
+  padding-inline: 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $additional-text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    color: $main-text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $additional-text-color;
+  }
+
+  &__numbers {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__buy {
+    margin-top: 16px;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    color: white;
+    font-size: 14px;
+    font-weight: 700;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $color-content;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+    border: 1px solid $color-content;
+    box-sizing: border-box;
+    line-height: 16px;
+  }
+
+  &__link:hover {
+    background-color: #fff;
+    border: 1px solid $color-content;
+    color: #00acdc;
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The hover effect on the card scales it by 1.2 times. This might affect the layout by causing overlap or other layout issues. Consider adjusting margins or layout to prevent any overlap or layout issues.